tests/data/setup.py

```diff
@@ -14,6 +14,9 @@
 requirements = [
     'click>=5.0.0',
     'mock>=1.3.0',
+    'invenio-db[versioning]>=1',
+    'invenio-db[versioning,mysql]>=1',
```

Comment:
IMHO it's more complicated in real life. This case should never happen since it's obvious to fix. There might be however following case:

```
requirements = [
    'invenio-db[versioning]>=1',
]

extras_require = {
     'mysql': [
          'invenio-db[mysql]>=1',
     ],
}
```

```console
$ requirements-builder -e mysql ...
invenio-db[versioning,mysql]>=1
```